Documentation/Releases/solr-release-11-1.rst

```diff
@@ -0,0 +1,67 @@
+.. ==================================================
+.. FOR YOUR INFORMATION
+.. --------------------------------------------------
+.. -*- coding: utf-8 -*- with BOM.
+
+.. include:: ../Includes.txt
+
+
+.. _releases-11-1:
+
+============================
+Apache Solr for TYPO3 11.1.0
+============================
+
+We are happy to release EXT:solr 11.1.0.
+The focus of this release has been on URL and SEO optimizations.
```

Comment:
solarium upgrade to

Reply:
Added the Solarium upgrade and it's breaking changes to the "New in this release" section